src/Controls/tests/TestCases.HostApp/MauiProgram.cs

```diff
@@ -110,7 +110,7 @@ protected override Window CreateWindow(IActivationState activationState)
 #endif
 
 #endif
-
+			NoCaretEntryHandler.RemoveCaret();
```

Comment:
Shouldn't we just create the Handler for NoCaretEntry and register it like we do for CollectionView. ?

Then we can create another issue to replace existing tests with local:NoCaretEntry.

Reply:
Updated and now we create a Handler and register it in MauiProgram. Also, added a property IsCursorVisible, to modify it in case necessary.